From 813895a25bec5c9193ffc096f15f40d271af04d2 Mon Sep 17 00:00:00 2001
From: Chelsea Mafrica <chelsea.e.mafrica@intel.com>
Date: Tue, 12 Nov 2019 11:11:43 -0800
Subject: [PATCH] obs-studio: change default config to ffmpeg

Change default config to ffmpeg advanced with webm as default format.

Signed-off-by: Chelsea Mafrica <chelsea.e.mafrica@intel.com>
---
 UI/window-basic-main.cpp | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/UI/window-basic-main.cpp b/UI/window-basic-main.cpp
index 5cd8f8de254f..9a1d09f89421 100644
--- a/UI/window-basic-main.cpp
+++ b/UI/window-basic-main.cpp
@@ -1619,7 +1619,7 @@ bool OBSBasic::InitBasicConfigDefaults()
 
 	/* ----------------------------------------------------- */
 
-	config_set_default_string(basicConfig, "Output", "Mode", "Simple");
+	config_set_default_string(basicConfig, "Output", "Mode", "Advanced");
 
 	config_set_default_bool(basicConfig, "Stream1", "IgnoreRecommended",
 				false);
@@ -1657,7 +1657,7 @@ bool OBSBasic::InitBasicConfigDefaults()
 	config_set_default_uint(basicConfig, "AdvOut", "VodTrackIndex", 2);
 	config_set_default_string(basicConfig, "AdvOut", "Encoder", "obs_x264");
 
-	config_set_default_string(basicConfig, "AdvOut", "RecType", "Standard");
+	config_set_default_string(basicConfig, "AdvOut", "RecType", "FFmpeg");
 
 	config_set_default_string(basicConfig, "AdvOut", "RecFilePath",
 				  GetDefaultVideoSavePath().c_str());
@@ -1672,7 +1672,7 @@ bool OBSBasic::InitBasicConfigDefaults()
 	config_set_default_bool(basicConfig, "AdvOut", "FFOutputToFile", true);
 	config_set_default_string(basicConfig, "AdvOut", "FFFilePath",
 				  GetDefaultVideoSavePath().c_str());
-	config_set_default_string(basicConfig, "AdvOut", "FFExtension", "mp4");
+	config_set_default_string(basicConfig, "AdvOut", "FFExtension", "webm");
 	config_set_default_uint(basicConfig, "AdvOut", "FFVBitrate", 2500);
 	config_set_default_uint(basicConfig, "AdvOut", "FFVGOPSize", 250);
 	config_set_default_bool(basicConfig, "AdvOut", "FFUseRescale", false);
-- 
2.45.1

